hieradata/clients/controller.trusted.ci.jenkins.io.yaml

```
@@ -181,8 +181,50 @@ profile::jenkinscontroller::jcasc:
           useAsMuchAsPossible: true
           credentialsId: "azure-jenkins-user"
           usePrivateIP: true
+        - name: "win-2019-amd64-maven-17"
+          description: "Windows 2019 Maven-17"
+          imageDefinition: jenkins-agent-windows-2019-amd64
+          os: "windows"
+          os_version: "2019"
+          launcher: "ssh"
+          location: "East US 2"
+          instanceType: Standard_D8ads_v5 # 8 vCPUS / 32 Gb
+          ephemeralOSDisk: true
+          spot: true
+          architecture: amd64
+          labels:
+            - docker-windows
+            - win-2019
+            - windows-2019
+            - win-2019-amd64-maven-17
```

```suggestion
          labels:
            - win-2019-amd64-maven-17
```

Otherwise using the label `windows` (for instance) can result in a random JDK used (may be 11, or 17 or 21)